keps/sig-network/3705-cloud-node-ips/README.md

```diff
@@ -208,55 +161,19 @@ annotation, and the cloud provider will see it, realize that the node
 IP request can't be fulfilled, log an error, and leave the node in the
 tainted state.
 
-It makes sense to have the same behavior if the user passes a "new"
-(eg, dual-stack) `--node-ip` value to kubelet, but the cloud provider
-does not recognize the new syntax and thus can't fulfil the request.
+It makes sense to have the same behavior if the user passes a
+dual-stack `--node-ip` value to kubelet, but the cloud provider does
+not recognize the new syntax and thus can't fulfil the request.
 Conveniently, we can do this just by passing the dual-stack
 `--node-ip` value in the existing annotation; the old cloud provider
 will try to parse it as a single IP address, fail, log an error, and
 leave the node in the tainted state, which is exactly what we wanted
 it to do if it can't interpret the `--node-ip` value correctly.
 
-Therefore, we do not _need_ a new annotation for the new `--node-ip`
+Therefore, we do not need a new annotation for the new `--node-ip`
```

Comment:
You removed the part about the fact that annotation name is `alpha.kubernetes.io/provided-node-ip` - what we're going to do with it?

Reply:
As you may have noticed later, that text is still in the "Alternatives" section.

I'm not planning to do anything about the annotation as part of _this_ KEP. I suspect I'm going to end up doing a new KEP with all of the leftover parts of this and #1665.